styx-common/src/main/java/com/spotify/styx/testdata/TestData.java

```diff
@@ -172,6 +173,7 @@
           .runningTimeout(Duration.parse("PT20H"))
           .retryCondition("#tries<2")
           .env("foo","bar")
+          .deploymentTime(Instant.ofEpochSecond(0))
```

Comment:
```suggestion
          .deploymentTime(WORKFLOW_DEPLOYMENT_TIME)
```
Maybe you can create a constant and use it on other test too